dist/exponential.go

```diff
@@ -229,8 +218,26 @@ func (e Exponential) Survival(x float64) float64 {
 	return math.Exp(-e.Rate * x)
 }
 
-// UnmarshalParameters implements the ParameterMarshaler interface
-func (e *Exponential) UnmarshalParameters(p []Parameter) {
+// Variance returns the variance of the probability distribution.
+func (e Exponential) Variance() float64 {
+	return 1 / (e.Rate * e.Rate)
+}
+
+// getParameters implements the parameterMutator interface
+func (e Exponential) getParameters(p []Parameter) []Parameter {
```

Comment:
Unless it collides with an existing name `parameters`. In which case add to TODO to name it `Parameters` when it is exported.